Source/Scene/Batched3DModel3DTileContentProvider.js

```diff
@@ -122,6 +122,9 @@ define([
         this.state = Cesium3DTileContentState.LOADING;
 
         loadArrayBuffer(this._url).then(function(arrayBuffer) {
+            if (that.isDestroyed()) {
+                throw new Error('tileset is destroyed');
```

Comment:
We only throw `DeveloperError` or `RuntimeError`, not `Error`; see https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/CodingGuide/README.md#throwing-exceptions

In this case, there was an outstanding request after the tileset was destroyed, right?  That is not an exceptional case.  Can we instead just silently destroy the tile?